onnxruntime/python/tools/transformers/models/bert/eval_squad.py

```diff
@@ -190,7 +197,11 @@ def main():
         tokenizer.model_max_length = sequence_length
         tokenizer.doc_stride = min(sequence_length // 2, 128)
 
-        ort_model, onnx_path = load_onnx_model(pretrained_model_name, args.onnx, args.use_gpu)
+        use_gpu = False
+        if args.provider is not ("CPUExecutionProvider" or None):
```

Comment:
What does it mean? This line of code looks very specious.

Reply:
In what sense?

Reply:
```
 if args.provider is not ("CPUExecutionProvider" or None):
```
Would you please provide some expected input/output of this statement? I don't quite understand what it is trying to do. For example, if args.provider  is None, would you expect the expression being evaluated to True or False?

Reply:
The expectation is if we're not using CPUExecutionProvider, we're using a gpu based execution provider, which gets checked when run later when we load the model if the provider isn't build into onnxruntime.  

The function load_onnx_mode, will error the user has entered an incorrect or unsupported provider. This is used to trigger everything but an empty provider string (None) and "CPUExecutionProvider" to trigger the gpu flag.

Is there some other way you want to see this change?

Reply:
![image](https://user-images.githubusercontent.com/856316/202290956-35afad7a-fc7b-4208-8573-10a28c4185e4.png)

Reply:
I've removed the none to clear this up.